CDS/scripts/js_utils/youtube/youtube-upload-lib.js

```diff
@@ -131,24 +132,42 @@ function addPosterImageIfExists(connection, videoId, youtubeClient, account) {
 
 function uploadToYoutube(connection) {
     return new Promise((resolve, reject) => {
+        console.log("DEBUG: getting auth client");
```

Comment:
could we move this to the [youtube-auth module](https://github.com/guardian/content_delivery_system/blob/master/CDS/scripts/js_utils/youtube/youtube-auth.js) to make it more contextual?